package.json

```diff
@@ -8,7 +8,7 @@
     "jsplumb": "2.13.2",
     "moment": "^2.29.4",
     "pica": "6.0.0",
-    "plotly.js": "^1.58.4",
+    "plotly.js": "^2.24.3",
```

Comment:
We should be able to bump to v2 safely. Our integration only makes simple plots and the breaking changes in v2 were limited. I manually verified that with v2, we can still create plots in wikis.